app.js

```diff
@@ -1,23 +1,16 @@
 const express = require('express')
-
-const { getTopics, getAllApi, getArticleId } = require('./controllers/controller')
-
-const {forCustomErrors, forPgErrors} = require('./error');
-
+const { getTopics, getAllApi, getAllArticles } = require('./controllers/controller')
 
 
 const app = express()
+
 app.use(express())
+
 app.get('/api/topics', getTopics)
 
 app.get('/api', getAllApi)
 
-app.get('/api/articles/:article_id', getArticleId)
-
-
-app.use(forCustomErrors);
-app.use(forPgErrors);
-
+app.get('/api/articles', getAllArticles)
 
```

Comment:
I can't see your error handlers in this commit but do make sure that any errors that aren't psql or custom are also caught in a final default error handler. (You will need this if the catch block for this endpoint is to have any effect!) An example of the error handling middleware chain can be found in the notes here https://notes.northcoders.com/courses/js-back-end/error-handling#4-refactoring-to-individual-error-handling-blocks

Reply:
Will fix this up now. Thank you for the feedback